cypress/integration/parser.unit.test.ts

```diff
@@ -0,0 +1,66 @@
+import { parseQueryStack } from "../../src/utils/parsers";
+import { expect } from "chai";
+//FIXME cannot use prettier for tests... 
+// import { format } from "prettier";
+
+describe("Unit tests", () => {
+  describe("Query Parser", () => {
+    it("should auto complete queries generated ending on a Root Node with no fields selected", () => {
+      let array = ["allFilms", "edges", "node"];
```

Comment:
Variables are never reassigned a value in this test file. Is everyone on board to use const instead of let?